cmd/ui.go

```diff
@@ -401,3 +408,17 @@ func yamlPrint(w io.Writer, v interface{}) error {
 	}
 	return nil
 }
+
+func getExtensionsDescription(exts []*ext.Extension, leftPad, padStart int) []string {
```

Comment:
Maybe `formatExtensionsDescription` ?

Reply:
:thinking: This is a bit more involved than what formatting would imply, so I think I prefer keeping it as `get*`, if you don't mind.

Otherwise, please state your case. :slightly_smiling_face:

Reply:
Following your argument about `get` being too generic below, I also renamed this to `formatExtensionsDescription` in 37b0e1e95.